src/black/linegen.py

```diff
@@ -197,6 +197,24 @@ def visit_decorators(self, node: Node) -> Iterator[Line]:
             yield from self.line()
             yield from self.visit(child)
 
+    def visit_power(self, node: Node) -> Iterator[Line]:
+        for idx, leaf in enumerate(node.children[:-1]):
+            next_leaf = node.children[idx + 1]
+            if (
+                isinstance(leaf, Leaf)
+                and leaf.type == token.NUMBER
+                and next_leaf.type == syms.trailer
+                # Integers prefixed by 0x are represented under the hood in just
```

Comment:
I'm not sure why this comment is relevant; at this point the ints are represented as strings.